stock_picking_invoice_link/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.remove_tables_fks(env.cr, ["stock_move_invoice_line_rel"])
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE stock_move_invoice_line_rel rel
```

Comment:
I'm afraid you can't do it this way due to 2 main reasons:

- It's destructive: you are removing the traceability in case something is needed, so we always try to preserve previous values when feasible, so a re-query can be done in case of need.
- The worst one: m2m tables are very special in the ORM, and the FK constraints and the rest of the table specification is only done at table creation. With this, you are having a dangling table with no consistency (as you have removed the FK).

What you need to do is:

- In pre-migration, rename the table `stock_move_invoice_line_rel`, and drop FK the same as here.
- In post, do an INSERT INTO in the new created table by the ORM process with the proper values.